helm/thingsboard/templates/node.yaml

```diff
@@ -53,6 +53,10 @@ spec:
           command: ['sh', '-c',
             'until pg_isready -h {{ include "thingsboard.pgpoolservicename" . }} -p {{ index .Values "postgresql-ha" "pgpool" "containerPort" }} -t {{ .Values.postgresInitDB.job.check.timeout }};
             do echo waiting for database; sleep 2; done;']
+        - name: check-db-queue-ready
+          image: postgres:{{ index .Values "postgresql-ha" "postgresqlImage" "tag" }}
```

Comment:
Why do we need the '"'? Is it because of '-' in the path? I think this 'ha' suffix is also some parameter, so we might want to use it here to avoid surprises, please check.

Reply:
It is because of "-" that we put quotes.

> I think this 'ha' suffix is also some parameter, so we might want to use it here to avoid surprises, please check.

I don't understand your suggestion

Reply:
I think I confused this 'ha' with something else, sorry for the noise!